tests/ops/op_math/test_pow_op.py

```diff
@@ -605,7 +605,7 @@ def test_matrix_against_shortcut_torch(self, z):
 
     @pytest.mark.tf
     @pytest.mark.parametrize("z", (2, -2, 1.23, -0.5))
-    def test_matrix_against_shortcut_jax(self, z):
+    def test_matrix_against_shortcut_tf(self, z):
```

Comment:
oops...

Reply:
I think there are quite a few other things along these lines and I usually don't mind, but I actually found this while (I think..) changing `qml.matrix`. Seemingly unrelated, but this test kept failing! It turns out it's because it wasn't supposed to be running in that environment at all. Why it only failed after making some unrelated change, I will never know.